modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/climax.dm

```diff
@@ -0,0 +1,161 @@
+#define CLIMAX_VAGINA "Vagina"
+#define CLIMAX_PENIS "Penis"
+#define CLIMAX_BOTH "Both"
+
+#define CLIMAX_ON_FLOOR "On the floor"
+#define CLIMAX_IN_OR_ON "Climax in or on someone"
+
+/mob/living/carbon/human/proc/climax(manual = TRUE)
+	if (CONFIG_GET(flag/disable_erp_preferences))
+		return
+
+	if(!client?.prefs?.read_preference(/datum/preference/toggle/erp/autocum) && manual != TRUE)
```

Comment:
Is there a reason it's `manual != TRUE` rather than just `!manual`?

```suggestion
	if(!client?.prefs?.read_preference(/datum/preference/toggle/erp/autocum) && !manual)
```

Reply:
Old code leftover.